README.md

```diff
@@ -39,8 +38,8 @@ this fashion.
 
 License and Copyright
 ---------------------
-Copyright (C) 2015-2017 Pelagicore AB  
-Copyright (C) 2017-2018 Luxoft Sweden AB
+Copyright &copy; 2015-2017 Pelagicore AB  
+Copyright &copy; 2017-2019 Luxoft Sweden AB
```

Comment:
No need for the period, just type the year for Luxoft.

Reply:
I don't really know; no one has ever made a decision. Either way seems to be good.

Reply:
I had a conversation with Therese regarding this. Our license will be covering for more than 70 years. You basically dont even have to update the years..